src/app/containers/Figure/index.jsx

```diff
@@ -7,6 +7,14 @@ import LazyLoad from 'react-lazyload';
 import Copyright from '../Copyright';
 import Caption from '../Caption';
 import { RequestContext } from '../../contexts/RequestContext';
+import {
+  NestedGridParentLarge,
+  NestedGridParentMedium,
+  NestedGridParentSmall,
+  NestedGridItemChildSmall,
+  NestedGridItemChildMedium,
+  NestedGridChildItemLarge,
```

Comment:
```suggestion
  NestedGridItemChildLarge,
```
would it be a good idea to make the naming consistent in this PR? this weird inconsistency could be a little confusing and I clearly missed it in my work 😬

Reply:
Alright will fix that in the styledGrid file